Src/libCZI/Doc/using_libCZI.markdown

```diff
@@ -226,4 +226,17 @@ and it converts the display-settings we got from the document's metadata into th
 The complete operation is depicted here:
 
 ![multi-channel-composite from a (scaled) tile-composite](ScalingSingleChannelTileAccessor1.png "scaled_multi-channel-composite1")
-@image latex ScalingSingleChannelTileAccessor1.png "multi-channel-composite from a (scaled) tile-composite"
\ No newline at end of file
+@image latex ScalingSingleChannelTileAccessor1.png "multi-channel-composite from a (scaled) tile-composite"
+
+## stream objects
+
+All input/output operations in libCZI are done through stream objects. Stream objects are used by the CZIReader to access the data in a CZI-file.
```

Comment:
Optional Markdown convention:
One sentence per line

Reply:
Specifically the start of a new sentence (without any additional spacing requirements), gets its own line

Reply:
Good idea, thanks, appreciated - I'll consider this in the future.